life_expectancy/cleaning.py

```diff
@@ -0,0 +1,97 @@
+from pathlib import Path
+import argparse
+import pandas as pd
+
+
+CURRENT_DIR = str(Path(__file__).parent)
+LIFE_EXPECTANCY_DATA_PATH = CURRENT_DIR + "/data/eu_life_expectancy_raw.tsv"
+OUTPUT_DATA_PATH = CURRENT_DIR + "/data/pt_life_expectancy.csv"
+
+
+def main(region: str = "PT") -> None:
+    """Loads the life expectancy, cleans and saves it as a csv file"""
+    life_expectancy_df = load_data()
+    life_expectancy_df_cleaned = clean_data(life_expectancy_df, region)
+    save_data(life_expectancy_df_cleaned)
+
+
+def load_data() -> pd.DataFrame:
+    """Loads the life expectancy TSV to a pandas DataFrame"""
+    return pd.read_table(LIFE_EXPECTANCY_DATA_PATH)
+
+
+def clean_data(
+    life_expectancy_df: pd.DataFrame, region: str = "PT"
+) -> pd.DataFrame:
+    """Cleans the data in the life expectancy DataFrame"""
+    return (
+        life_expectancy_df
+        .pipe(_split_column_into_several)
```

Comment:
Very nice the dataframe.pipe() method! I didn't know about it. I think I will use it in my code too.

Reply:
I found out about it a few weeks ago! It's really cool to create a DataFrame pipeline